app/static/ringing_room.js

```diff
@@ -967,6 +970,11 @@ $(document).ready(function () {
                     title: "Double Norwich Court Bob Major",
                     url: "Double_Norwich_Court_Bob_Major",
                 },
+
```

Comment:
Will this still be an issue if there's no auto-change of peal speed on tower size adjustment?

Reply:
I don't think so, because I think that the automatic peal speed adjustment is the only thing that directly uses the ratio between the last/new tower sizes.  Both signals show the same tower size change, so receiving e.g. `6 -> 8` twice would cause the peal speed to be updated twice.  I don't believe anything else uses any information about the previous tower size.